go/consensus/tendermint/apps/beapoch/state/state_scrape.go

```diff
@@ -0,0 +1,62 @@
+package state
+
+import (
+	"context"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/scrape"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+	"github.com/oasisprotocol/oasis-core/go/common/keyformat"
+	abciAPI "github.com/oasisprotocol/oasis-core/go/consensus/tendermint/api"
+	epochtime "github.com/oasisprotocol/oasis-core/go/epochtime/api"
+)
+
+var scrapeStateKeyFmt = keyformat.New(0x42)
+
+// RoundState is a SCRAPE round state.
+type RoundState int
+
+const (
+	StateInvalid RoundState = iota
```

Comment:
Since this is serializable I would rather we use explicit values instead of `iota`.